packages/speechd/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/brailcom/speechd
+TERMUX_PKG_DESCRIPTION="Common high-level interface to speech synthesis"
+TERMUX_PKG_LICENSE="GPL2, GPL3 and LGPL"
```

Comment:
Use comma-separated (no "and") SPDX license identifiers here.

See https://github.com/termux/termux-packages/blob/master/scripts/lint-packages.sh#L18-L36.

Reply:
Thank you for the suggestion; modified as requested.